openedx/core/djangoapps/notifications/models.py

```diff
@@ -18,7 +18,7 @@
 User = get_user_model()
 log = logging.getLogger(__name__)
 
-NOTIFICATION_CHANNELS = ['web', 'push', 'email']
+NOTIFICATION_CHANNELS = ['web', 'push', 'email', 'email_cadence']
```

Comment:
`email_cadence` does not seem like a part of  NOTIFICATION_CHANNELS i.e email is channel but email_cadence is not, can we implement this feature without adding it as a new channel .